mountpoint-s3-client/src/s3_crt_client/put_object.rs

```diff
@@ -252,3 +252,10 @@ impl PutObjectRequest for S3PutObjectRequest {
         })
     }
 }
+
+impl S3PutObjectRequest {
+    /// The number of bytes written to this request so far.
+    pub fn bytes_written(&self) -> u64 {
```

Comment:
Do we plan to expose this function at `PutObjectRequest` interface so that other variants of `PutObject` can report this too? If that's the case then we probably want to add some TODO.

Reply:
Added TODO